lib/airbrake/rake/tasks.rb

```diff
@@ -67,8 +67,8 @@
 
       # Avoid loading the environment to speed up the deploy task and try guess
       # the initializer file location.
-      if initializer.exist?
-        load(initializer) unless Airbrake[:default]
+      if initializer.exist? && Airbrake[:default].is_a?(Airbrake::NilNotifier)
```

Comment:
> Check for NilNotifier is hacky

It is. This whole expression is a hack to make the task run faster.

> I have no idea why you load initialized only when default notifier is NilNotifier

That means Airbrake wasn't configured yet. To send deploy info, we need a configured notifier.

This hack is needed to speedup the task. It used to works quite reliably, if we don't count the fact that it got broken in this release :)